addons/binding/org.openhab.binding.freebox/ESH-INF/thing/thing-types.xml

```diff
@@ -44,6 +44,20 @@
  				<required>false</required>
  			</parameter>
  
+            <parameter name="apiBaseUrl" type="text">
```

Comment:
Could you please run the formatter over the whole file, since now there is a mix of spaces and tabs in here?